pegasus/sites.v3/code.org/public/translate/frozen.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "Translate Frozen"
+nav: translate_nav
+theme: responsive
+---
+
+# Translating Code with Anna and Elsa
+Translations for the [Code with Anna and Elsa](https://studio.code.org/s/frozen) tutorial are done on Crowdin and translations for the video captions are done on Amara. They are both free to join and any volunteer can immediately start translating on either platform.
+
+## Getting set up
+
+1. [Click here](https://amara.org/en/auth/login/?next=/) to create an Amara account.
+1. [Click here to join our Code.org project](https://crowdin.com/project/codeorg) on Crowdin.
+1. From the [main project page](https://crowdin.com/project/codeorg), select your language.
+1. Note your language code in the URL, you'll need this for later. If it's a 4 letter code like "zh-TW" note that the code you'll use later will be in the format "zhTW".
+
+<img src="/images/fit-500/translate-find-code.jpg" style="max-width: 100%;"> 
+
+## Translating the tutorial progression
+The following links all lead directly to the Spanish tutorial. Replace the "es" in the URL with the language code you got earlier. Your translations will be updated on the site at the end of each week. You can watch this [demo video](/files/crowdin.swf) for how to use Crowdin.
+
+1. [tutorial title](https://crowdin.com/translate/codeorg/63/enus-es#68955)
```

Comment:
title: `[here](https://crowdin.com/translate/codeorg/63/enus-es#23533) and [here](https://crowdin.com/translate/codeorg/40/enus-es#24790)`